index.html

```diff
@@ -52,6 +52,21 @@
 <h2>Articles and blog entries</h2>
 <p>To add a link to an article or blog entry, follow the README guidelines and submit a pull request.</p>
 
+<div class="submission">
+<p class="date"><time datetime="2015-10-26">26th October 2015</time></p>
+<h3>
+<a href="http://www.smashingmagazine.com/2015/10/dealing-with-loud-silent-burnout/">Dealing with loud and silent burnout</a>
+</h3>
+<p class="author">
+<i>by</i> <a href="https://www.twitter.com/retinart">Alexander Charchar</a>
+</p>
+<p>We’ve all experienced that burnout moment. It’s that moment when we’ve got nothing left to give but keep trying anyway, when we’re left without much more than a shell to live in and motions to go through.</p>
+
+<p>We’re fried and broken and wish desperately for our work to make sense, for our energy to come back, for things to be fun and as they were. In such moments all we want is for our work to feel like our work and not like torture.</p>
+
+<p>“Are you all right? You’re a little gray,” our friends and loved ones say with concern. “You should get some rest.”</p>
+</p>
```

Comment:
I'm sorry but it seems I can't see the obvious.
What tag needs to be removed? The whole part from line 55 to 68 or just the surrounding `<div>`?

Thought I'd followed the guidelines. :frowning: